src/components/layout/Slider.svelte

```diff
@@ -41,7 +41,7 @@
 		let progress = progressPercentOnSliderGrab + 100 * xDiff / sliderElem.offsetWidth;
 		if (progress < 0) progress = 0;
 		if (progress > 100) progress = 100;
-		progressPercent = snapProgress(progress);
+		showDots ? progressPercent = snapProgress(progress) : progressPercent = Math.round(progress);
```

Comment:
This logic should probably be in the snapProgress function, since it is used elsewhere in the code. 

```
function snapProgress(percent) {
                 if (!showDots) return Math.round(percent);
...
	}
```

Reply:
My rationale was that if I click in the middle of the slider at first, I probably want 50x leverage, for example. So having Snap on click might actually make sense, if you don't agree I will push the updated version.

Reply:
I think it's better to have consistency with the snapping, whether it's clicked or dragged.